ansible/files/sonic_lab_devices.csv

```diff
@@ -1,10 +1,10 @@
-Hostname,ManagementIp,HwSku,Type,Protocol
-dev-msn2700-01,10.251.0.188/23,Mellanox-2700,DevSonic,
-dev-7260-10,10.251.0.13/23,Arista-7260QX-64,FanoutLeaf,
-dev-7260-11,10.251.0.234/23,Arista-7260QX-64,FanoutRoot,
-dev-acs-serv-01,10.251.0.245/23,TestServ,Server,
-pdu-1,192.168.9.2,Apc,Pdu,snmp
-pdu-2,192.168.9.3,Sentry,Pdu,snmp
-console-1,192.168.10.1/23,Cisco,ConsoleServer,ssh
-console-2,192.168.10.2/23,Sonic,ConsoleServer,ssh
-management-1,192.168.10.3/23,Sonic,MgmtTsToRRouter,
+Hostname,ManagementIp,HwSku,Type,Protocol,Os
+dev-msn2700-01,10.251.0.188/23,Mellanox-2700,DevSonic,,sonic
+dev-7260-10,10.251.0.13/23,Arista-7260QX-64,FanoutLeaf,,sonic
+dev-7260-11,10.251.0.234/23,Arista-7260QX-64,FanoutRoot,,eos
+dev-acs-serv-01,10.251.0.245/23,TestServ,Server,,ubuntu
+pdu-1,192.168.9.2,Apc,Pdu,snmp,
+pdu-2,192.168.9.3,Sentry,Pdu,snmp,
+console-1,192.168.10.1/23,Cisco,ConsoleServer,ssh,sonic
+console-2,192.168.10.2/23,Sonic,ConsoleServer,ssh,cisco
```

Comment:
How about use `ConsoleServerSonic` as device type to mark it as Sonic? (Just like what we do for `FanoutLeaf` and `FanoutLeafSonic`)

Reply:
As discussed offline, add a separate column is better choice.